### ISSUE TITLE
Training does not work with newest commit and pytorch 1.3

### ISSUE THREAD

**Issue by LeonhardFeiner** [facebookresearch/SparseConvNet#134]
Unfortunately, the newest version of SparseConvNet does not work for me.
Previously I trained ScanNet with an older [version](https://github.com/facebookresearch/SparseConvNet/commit/16e4df34eeaf1967cd4d9a13f5c0393535b4a882) and Pytorch 1.2 and it worked well.
After 256 epochs the IoU increases significantly.
```
256 Train loss 0.28119953428253985 MegaMulAdd= 885.8841321397546 MegaHidden 2.9489331973559962 time= 321.5200023651123 s
256 1 Val MegaMulAdd= 909.9527086519824 MegaHidden 3.062086202643172 time= 339.2394509315491 s
evaluating 69817068 points...
classes          IoU
----------------------------
wall          : 0.759   (14321876/18872018)
floor         : 0.928   (12079402/13018232)
cabinet       : 0.474   (1279399/2701789)
bed           : 0.724   (1386502/1915337)
chair         : 0.815   (3226901/3957838)
sofa          : 0.773   (1141003/1475821)
table         : 0.556   (1362224/2449105)
door          : 0.459   (1239012/2700720)
window        : 0.488   (1287669/2640789)
bookshelf     : 0.630   (1195862/1897787)
picture       : 0.195   ( 67442/345786)
counter       : 0.446   (161645/362453)
desk          : 0.442   (556192/1259301)
curtain       : 0.538   (928792/1726036)
refrigerator  : 0.515   (158647/308288)
shower curtain: 0.502   (145806/290571)
toilet        : 0.836   (167472/200339)
sink          : 0.455   ( 77430/170171)
bathtub       : 0.737   (176135/238916)
otherfurniture: 0.306   (881094/2875920)
mean IOU 0.5788562547685823
```

When using the newest [version](https://github.com/facebookresearch/SparseConvNet/commit/a948af5889bf420cdbda56ffea82b0ba9b06c125) and Pytorch 1.3 on Cudatoolkit 10.0 it does not work.
```
256 Train loss 2.168078747662631 MegaMulAdd= 879.1651310443815 MegaHidden 2.936921926345609 time= 124.86485457420349 s
256 1 Val MegaMulAdd= 906.6917951013215 MegaHidden 3.0561749427312774 time= 184.37045073509216 s
evaluating 69817068 points...
classes          IoU
----------------------------
wall          : 0.310   (15705998/50623861)
floor         : 0.000   (     0/12378071)
cabinet       : 0.000   (     0/2013704)
bed           : 0.000   (     0/1670865)
chair         : 0.000   (     0/3666059)
sofa          : 0.000   (     0/1275482)
table         : 0.000   (     0/1836381)
door          : 0.000   (     0/2120926)
window        : 0.000   (     0/2086159)
bookshelf     : 0.000   (     0/1496618)
picture       : 0.000   (     0/288553)
counter       : 0.000   (     0/271391)
desk          : 0.000   (     0/963940)
curtain       : 0.000   (     0/1425386)
refrigerator  : 0.000   (     0/251429)
shower curtain: 0.000   (     0/259622)
toilet        : 0.000   (     0/190385)
sink          : 0.000   (     0/156174)
bathtub       : 0.000   (     0/217156)
otherfurniture: 0.000   (     0/2349562)
mean IOU 0.01551244580100281
```
The same happens for my own project.

**Comment by btgraham** [comment]
After 4 epochs I see it starting to learn. Does it explode later on?

4 Train loss 1.4337332538656287 MegaMulAdd= 879.0726951140716 MegaHidden 2.9264693288925896 time= 89.71132254600525 s
4 1 Val MegaMulAdd= 916.5389126153846 MegaHidden 3.093964205128205 time= 91.53541088104248 s
evaluating 49540568 points...
classes          IoU
----------------------------
wall          : 0.607   (10584718/17432575)
floor         : 0.886   (8806195/9944264)
...
mean IOU 0.11522306153521769

**Comment by LeonhardFeiner** [comment]
It did not learn at all and seemed to be constant
I built pytorch 1.3 on my own because of cuda 10.1 was not available at my server yet and either 10.1 or 9.2 is prerequiary for installing it by conda.
I'm not sure about the reason for that issue right now, but using cudatoolkit 9.2, pytorch 1.3 (installed by conda) and the newest version of sparseconvnet works.

I think that torchvision installed a second version of pytorch in the same environment and I didn't recognize it. I assume that this was the reason for learning nothing.
